.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
-repos:
-- repo: https://github.com/pycqa/flake8
-  rev: 4.0.1
-  hooks:
-  - id: flake8
-- repo: https://github.com/omnilib/ufmt
-  rev: v1.3.0
-  hooks:
-    - id: ufmt
-      additional_dependencies:
-        - black == 21.4b2
-        - usort == 0.6.4
+repos:

```

Comment:
Ahh, our internal linter pointed it out. You've got dos style line breaks (`\r\n`) rather than unix style line breaks (`\n`). Could you please switch to the latter?